src/monet.js

```diff
@@ -177,6 +177,22 @@
         return listFindC(l, fn).run()
     }
 
+    function listContainsC(l, val) {
+        if (l.isNil) {
+            return Return(false)
+        }
+        var h = l.head()
+        return h === val ?
```

Comment:
one can always use our or immutable's `is()`:

```
arr.some(a => is(a, b))
```

Or we can make it curried so:

```
arr.some(is(a))
```

;)